### DISCUSSION TITLE
Format selection / Latexindent stopped working

### DISCUSSION THREAD

**Discussion by rubisco57** [James-Yu/LaTeX-Workshop#4442] [Q&A]
Up until recently, the format selection using latexindent worked without any problem. I think after update in Latex Workshop it started misbehaving. Consequentially, I updated VScode to 1.94.2 and latexindent to 3.24.4, latexindent-windows packages on texlive2024 to the latest versions. Still, upon formatting, only tab characters are replaced with `\t`.
My settings.json file only contains `"latex-workshop.formatting.latex": "latexindent"` related to the problem.
If you know a possible work around, I thank you for posting.

**Comment by sherlma23** [comment]
Recently, I found that format is not working. Then I change the version of latex-workshop to 10.3.0. It works now. You could try roll back  the version of latex-workshop to 10.3.0 or earlier.
